front/lib/api/assistant/configuration.ts

```diff
@@ -1317,7 +1324,7 @@ export async function createAgentActionConfiguration(
         id: websearchConfig.id,
         sId: websearchConfig.sId,
         type: "websearch_configuration",
-        name: action.name,
+        name: action.name || DEFAULT_WEBSEARCH_ACTION_NAME,
```

Comment:
nit: I was told in general ?? is more idiomatic than ||

Reply:
|| will catch `""` whereas ?? won't so it's a bit "safer again here" as we really want a name